api/src/services/friendshipService.ts

```diff
@@ -51,29 +52,31 @@ export default class FriendshipService {
     }
 
     public async createFriendship(
-        userId1: number,
-        userId2: number,
+        senderId: number,
+        receiverId: number,
         status: FriendshipStatus,
         actionUserId: number
     ) {
+        if (senderId == receiverId) throw new SelfFriendshipError();
+
         const areFriends = await ValidationUtils.areFriends(
             this.friendshipRepository,
-            userId1,
-            userId2
+            senderId,
+            receiverId
         );
 
         if (areFriends) throw new FriendshipAlreadyExistError();
 
         const friendship = await this.friendshipRepository.save({
-            UserId1: userId1,
-            UserId2: userId2,
+            UserId1: senderId,
```

Comment:
não seria legal mudar o nome dos atributos da entidade também?

Reply:
Acho que podemos deixar assim por enquanto. Se quisermos, podemos mudar na entidade para usar as relações many-to-many do typeorm depois e aí até facilita um pouco isso